### DISCUSSION TITLE
Two Factor required for bot ?

### DISCUSSION THREAD

**Discussion by Renaud-Dov** [Rapptz/discord.py#6585] [Q&A] [answered]
I tried to use MESSAGE_PERMISSIONS, but the bot return this : `403 Forbidden (error code: 60003): Two factor is required for this operation`

How can I fix the problem ?

**Comment by Zomatree** [comment, marked as answer]
The bot owner needs two factor authentication.

**Reply by Renaud-Dov** [reply]
ok thanks :+1:
